### PR TITLE
Update Seal hosting

### PR DESCRIPTION
**Pull request by robinjam**
Seal is no longer hosted on Heroku 🎉

### CODE REVIEW

**Review by ChrisBAshton — APPROVED** [review]

File: data/repos.yml [hunk 1]
```diff
@@ -1235,7 +1235,6 @@
 - repo_name: seal
   team: "#govuk-platform-security-reliability-team"
   management_url: https://dashboard.heroku.com/apps/govuk-seal
-  production_hosted_on: heroku
```
Comment (ChrisBAshton) [inline data/repos.yml:1238]: Is there a github actions equivalent in Dev Docs config?

Comment (ChrisBAshton) [reply data/repos.yml:1238]: (If not, should there be?)

Comment (robinjam) [reply data/repos.yml:1238]: Note that we determine whether or not a repo is an "app" by [looking at `production_hosted_on`](https://github.com/alphagov/govuk-developer-docs/blob/cc8d6ef03b100f7a6e3187b8c71f3672dd7a9d04/app/repo.rb#L49-L51), so assuming we agree that it no longer counts as an "app", we probably want it to be `nil`.

Comment (ChrisBAshton) [reply data/repos.yml:1238]: Fair 👍

**Review by ChrisBAshton — APPROVED** [review]

**Pull request merged**